wgpu-hal/src/gles/egl.rs

```diff
@@ -954,6 +972,10 @@ impl crate::Instance<super::Api> for Instance {
             })
         };
 
+        if !matches!(inner.srgb_kind, SrgbFrameBufferKind::None) {
+            unsafe { gl.enable(glow::FRAMEBUFFER_SRGB) };
+        }
```

Comment:
Can you please add a comment explaining this. Took me quite a while to figure out that this is needed because in GLES all GL_SRGB buffers do the expected conversions whereas on desktop GL one can opt out of it with this flag (wtf...)
(great catch btw.! :))